nodejs/week1/avg.js

```diff
@@ -0,0 +1,23 @@
+const myArgs = process.argv.slice(2);
+let numArray =[];
+let total = 0;
+let countOfNumbers = 0;
+
+if(myArgs.length === 0) {
+    console.log('Please specify some args the array is empty');
+} else {
+    myArgs.forEach(arg => {
+        const parsedArg = parseFloat(arg);
+        if(!isNaN(parsedArg)) {
```

Comment:
You could add a console log if parsedArg was not a number because if user has input string or special charcters